<launch>
	<param name="move_rate" value="20" />
  <param name ="maxForce" value="20" />

	<node name="dynamic_left" pkg="sub8_diagnostics" type="dynamic_left" />
</launch>
